src/Client/IO.hs

```diff
@@ -364,46 +253,95 @@ getFromTzbtcUStore
-      AlmostStorage{..} <- getTzbtcStorage @SomeTZBTCVersion contractAddr
-      let scriptExpr = encodeBase58Check . valueToScriptExpr . lPackValue $ key
-      fieldValueRaw <- getFromBigMap asBigMapId scriptExpr
-      case fieldValueRaw of
-        Left err -> case err of
-          TzbtcServantError (FailureResponse _ Response{..}) -> do
-            if responseStatusCode == notFound404 then pure Nothing
-            else throwM err
-          _ -> throwM err
-        Right veryRawVal -> do
-          rawVal <- throwLeft $ pure $ exprToValue @ByteString veryRawVal
-          fmap Just . throwLeft . pure $ lUnpackValueRaw rawVal
+  contractAddr <- getTzbtcContractAddress
+  -- Exact version does not matter for such low-level operation
+  AlmostStorage{..} <- getTzbtcStorage @SomeTZBTCVersion contractAddr
+  fieldValueRaw <- readBigMapValueMaybe (BigMapId asBigMapId) (lPackValue key)
+  case fieldValueRaw of
+    Nothing -> pure Nothing
+    Just packedValue -> Just <$> do
+      throwLeft $ pure $ lUnpackValue packedValue
 
 getTzbtcStorage
-  :: forall ver m. (Typeable ver, HasTezosRpc m) => Address -> m (AlmostStorage ver)
+  :: forall ver m. (HasTezosRpc m)
+  => Address -> m (AlmostStorage ver)
 getTzbtcStorage contractAddr = do
-  storageRaw <- getStorage $ formatAddress contractAddr
-  throwLeft $ pure $ exprToValue @(AlmostStorage ver) storageRaw
+  storageRaw <- getContractStorage contractAddr
+  either throwM (pure . fromVal) $
+    fromExpression @(Value (ToT (AlmostStorage ver))) storageRaw
 
 performTzbtcDeployment
-  :: (MonadIO m, MonadThrow m, HasTezosClient m, HasCmdLine m)
-  => (Bool -> ClientConfig -> IO Address) -> m ()
+  :: (MonadThrow m, HasTezosClient m, HasCmdLine m)
+  => m Address -> m ()
 performTzbtcDeployment deployAction = do
-  config@ClientConfig{..} <- throwLeft readConfig
-  v <- aeVerbose <$> lookupEnv
-  contractAddr <- liftIO $ deployAction v config
-  putTextLn $ "Contract was successfully deployed. Contract address: " <> formatAddress contractAddr
-  liftIO $ case ccContractAddress of
-    Just c -> putTextLn $ "Current contract address for alias 'tzbtc' in the tezos-client config is " <> formatAddress c <> "."
-    Nothing -> putTextLn "Right now there is no contract aliased as 'tzbtc' in tezos-client config."
+  contractAddr <- deployAction
+  printTextLn $ "Contract was successfully deployed. Contract address: " <> formatAddress contractAddr
+  mbTzbtcAddress <- resolveAddressMaybe (AddressAlias $ Alias "tzbtc")
+  case mbTzbtcAddress of
+    Just c -> printTextLn $ "Current contract address for alias 'tzbtc' in the tezos-client config is " <> formatAddress c <> "."
+    Nothing -> printStringLn "Right now there is no contract aliased as 'tzbtc' in tezos-client config."
+  res <- confirmAction "Would you like to add/replace alias 'tzbtc' with the newly deployed contract?"
+  case res of
+    Canceled -> pass
+    Confirmed -> rememberContract True contractAddr "tzbtc"
```

Comment:
```suggestion
  let contractAlias = "tzbtc"
  mbTzbtcAddress <- resolveAddressMaybe (AddressAlias $ Alias contractAlias)
  printTextLn $ case mbTzbtcAddress of
    Just c ->
      "Current contract address for alias '" <> contractAlias <>
      "' in the tezos-client config is " <> formatAddress c <> "."
    Nothing ->
      "Right now there is no contract aliased as '" <> contractAlias <>
      "' in tezos-client config."
  res <- confirmAction
    "Would you like to add/replace alias '" <> contractAlias <>
    "' with the newly deployed contract?"
  case res of
    Canceled -> pass
    Confirmed -> rememberContract True contractAddr contractAlias
```
Triggered by the `printStringLn`, but I think we should avoid copy-pasting the contract alias/name everywhere.